sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/di/SyncModule.kt

```diff
@@ -50,4 +53,14 @@ object SyncStoreModule {
     fun providesNativeLib(context: Context): SyncLib {
         return SyncNativeLib(context)
     }
+
+    @Provides
+    @SingleInstanceIn(AppScope::class)
```

Comment:
NIT: this one can be annotated instead of inside the module because is part of :impl where we have di